### ISSUE TITLE
Lint issue: Test files should not be imported.  ava/no-import-test-files

### ISSUE THREAD

**Issue by BBlackwo** [lassjs/lass#49]
On a brand new project (e.g. `npx lass my-new-project`) I get the following error when running `npm run lint`:

```
test/test.js:3:35
  ✖  3:35  Test files should not be imported.  ava/no-import-test-files
```

Is there a way I can fix this?

**Comment by niftylettuce** [comment]
Fix has been released in v2.0.4.

`npm i -g lass@latest`

<https://github.com/lassjs/lass/releases/tag/v2.0.4>

**Comment by niftylettuce** [comment]
Please ensure you are on v2.0.5 if you are using Node v8.x, I have fixed the issue and published new patch.
